app/app.py

```diff
@@ -106,8 +106,8 @@ def create_app(config_name=None) -> Application:
 
     # Set static folder location
     # TODO: Only turn on in dev environment
-    app["static_root_url"] = "/"
-    app.router.add_static("/", app["STATIC_ROOT"], show_index=True)
+    app["static_root_url"] = f'{app["URL_PATH_PREFIX"] or "/"}'
```

Comment:
This doesn't need to be wrapped in an f string.

Reply:
f_string_all_the_things.gif!